src/plugins/condition/components/Condition.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+<div id="conditionArea"
+     class="c-cs-ui__conditions"
+     :class="['widget-condition', { 'widget-condition--current': isCurrent }]"
+>
+    <div class="title-bar">
+        <span v-if="isDefault"
+              class="condition-name"
+        >Default
+        </span>
+        <span v-else
+              class="condition-name"
+        >[condition name]
+        </span>
+        <span v-if="isDefault"
+              class="condition-output"
+        >Output: false
```

Comment:
Should this output be false? Wouldn't `default` always be true?

Reply:
I was replicating the UX mockup. To be honest I'm not sure what output would mean in this context. Let's discuss.